x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_form/relevance_tuning_item_content/boosts/boosts.scss

```diff
@@ -0,0 +1,32 @@
+.boosts {
+  .boosts__header.euiPageContentHeader {
+    margin-bottom: 0;
+  }
+
```

Comment:
We can delete this now that we're not overriding `.euiPageContentHeader`

```suggestion
```

Reply:
we should also go ahead and remove the `boosts__header` className from the DOM probably

Reply:
fb5a13d